lib/validator.js

```diff
@@ -218,6 +218,10 @@ Specberus.prototype.getDocumentDate = function () {
             return false; // no need to look further
         }
     });
+
+    if (!this.docDate) {
+        return this.throw("The URL should be a TR document.");
+    }
```

Comment:
Thinking more about it, I think this should probably be handle when the rules are being checked (i.e. in `Specberus.prototype.extractMetadata` (see at the end of that comment)

```suggestion
```
```
        profileMetadata.rules.forEach(function (rule) {
            try {
              rule.check(self, function (result) {
                  if (result) {
                      for (var i in result) {
                          self.meta[i] = result[i];
                      }
                  }
                  done++;
                  self.sink.emit("done", this.name);
                  if (done === total)
                      self.sink.emit("end-all", util.buildJSONresult(errors, warnings, infos, self.meta));
              }.bind(rule));
            } catch (e) {
                self.throw(e.message);
            }
        });
```